core/services/ocrcommon/peer_wrapper.go

```diff
@@ -217,3 +217,7 @@ func (p *SingletonPeerWrapper) Close() error {
 		return err
 	})
 }
+
+func (p *SingletonPeerWrapper) Config() PeerWrapperConfig {
+	return p.config
+}
```

Comment:
Where is this used, I'm surprised its necessary

Reply:
this helps keep the evm specific stuff out of the ocr2 delegate. The p2p config is not chain specific, so we can access it from here as opposed to from chain.Config